index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <header id="header" >
+      <img id="header-img" src="Logo.png" alt="Logo">
+      <nav id="nav-bar"><a class="nav-link" href="#conocenos">Button 1</a> <a class="nav-link" href="#video">Button 2</a><a class="nav-link" href="#precios">Button 3</a></nav>
```

Comment:
Hi, @HFG43!! 👋
- Here you have made a good suggestion politely. Thats appreciable. 👍 

- [ ] Kindly use emoticon (😄) so that your review looks human-friendly.

- [ ] Kindly use reasoning `why` students need to consider your suggestion. Why do students need to use the `button` here? Use reasoning here and you can add any referral link from where students can get hints.

- [ ] I can see some grammatical errors, such as you have started the review with `Is` but it would be `It is`. Kindly use [Grammarly](https://app.grammarly.com/) so that there are no spelling mistakes and other grammatical mistakes. It is more professional to check the spelling while giving any suggestion in every aspect of our job and profession.